### ISSUE TITLE
Public way to indicate where/how to donate in Austin - making it easy for donators and orgs both

### ISSUE THREAD

**Issue by vpruthi** [open-austin/project-ideas#109]
### What problem are we trying to solve?

It is difficult to find what local organizations will accept (x) donations and if they do accept donations if they provide services including pickup.

### Who will benefit (directly and indirectly) from this project?
People want to donate however they also want it to be easy. Larger orgs like goodwill and salvation army aren't always the best options and have very specific requirements that aren't realized until the pickup happens. I've especially noticed this during pregnancy as well as Hurricane Harvey, as friends of mine who have passed down clothes/toys literally said they have left expensive and in good condition car seats, strollers, clothes just on the side as they got frustrated with trying to get them donated. And/or don't trust those corner area grocery drop off boxes.

Could start with a certain audience first (mothers and women), and then over time as others leverage this platform it can be expanded to other audiences if it works. Also, could start in Austin and allow it to be open so that other cities can build upon it.

--

### Where can we find any research/data available/articles?

Can totally find this if needed, but after taking a look at the current alternatives they are as follows:

a) each organization has their on site and info b) individuals rely on Craigslist or word of mouth. Aunt Bertha is another option however it lacks the ability to crowdsource info, add info around pickup / drop off details, and doesn't drill down by "what you want to donate" https://www.auntbertha.com/

### What help is needed at this time?

Help with dev/design. Just some initial thoughts but am totally open to other ideas - Building upon a city map, display orgs that accept donation. Indicate what items they accept, in what condition, which audience (if specific) that they serve, and donation/pickup/contact details. Use public data at first, and then allow organizations to insert their own location/info (via a pin... think yelp or google wave). Seeding the dataset with open data. Would be amazing to have a database / site of the donation centers across the country (long term vision but start with Austin).

### What are the next steps (validation, research, coding, design)?

Coding/Design. Release an MVP and let others utilize it to get feedback. Purpose is to have this remain public and grow by folks continuing to add to it over time.

----

#### Checklist for NEW ideas :baby:
Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).

- [x] Create this idea issue
- [x] Flesh out the who, where, and what questions above
- [ ] Start the conversation about this new idea on [Slack](https://openaustin-slackin.herokuapp.com/)!
- [ ] Respond with at least one update on this issue within the next month

#### Checklist for ACTIVE projects :fire:
Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:

- [x] Post an update at least once a month to this issue. Use [BASEDEF](https://github.com/open-austin/cheatsheet-basedef) for ideas, but it's ok even if your update is just "nothing new happened this month" or "we saw a small increase in traffic to our app this month". If there's no activity for two months, that's no problem, life happens. We'll just label this as `backlog` so others know you'll get back to it when you have the time. If nobody hears from you at all in more than two months, we may mark it as `abandoned` so that others can pick up this idea and run with it.
- [ ] Take 30 minutes to complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101) 
- [ ] Create a [GitHub repository](https://github.com/new) and Slack channel for work.
- [ ] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [ ] [Create issues](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#assignment--add-your-first-issue) to describe each task that you plan to do or need help with and how a contributor can get started on that task. You might start and stop a lot, so consider issues as your to-do list.

#### Checklist for FEATURED Projects :tada:
To have your project FEATURED on [Open-Austin.org](https://open-austin.org/), complete the following documentation. In past projects, well-documented featured projects have more contributions than other projects.

* [ ] In your README, link to the [Open Austin Community Participation Guidelines](https://docs.google.com/document/d/1OujyBccPpepUXSY5_nP3alzvd81WlCKSeSf_-sU5K-U/edit?usp=sharing) or write your own code of conduct.
* [ ] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
* [ ] [Create file: CONTRIBUTING.md](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md) so others know [how they can contribute](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/).
* [ ] Create an easily shareable project management artifact, like a [Civic Tech Canvas](https://cityofaustin.github.io/civic-tech-canvas/) or [Open Canvas](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/)

Once all of the above is complete,
- [ ] [Create an issue](https://github.com/open-austin/open-austin.github.io/issues/new) on the open-austin.github.io repo with the title `Add [my project] to projects page`. An Open Austin leader will review this issue and post your project :balloon:

If you get stuck at any point, feel free to reach out to the leadership team on Slack by adding @leadership to your message. We're here to help you make real changes to our city.

**Comment by werdnanoslen** [comment]
Hi @vpruthi! Thanks for starting this, I'm the projects lead at Open Austin. How it going? Is there anything I can help you with this project?

**Comment by vpruthi** [comment]
Thanks for checking in Andrew! I'd love your help in finding someone to
help on the coding side.

**Comment by werdnanoslen** [comment]
Sure thing, what  language are you coding in? Do you have a repo we could link to from here?

**Comment by vpruthi** [comment]
My background is business, non-dev :) Mateo recommended approaching the
open atx community to see if there was someone with a dev background whom I
could partner up with. Please let me know if you'd like any additional
details, I really appreciate it!

**Comment by werdnanoslen** [comment]
Gotcha, I think for now, we can do a low-fidelity iteration by just building the content. Could you start a Google Drive spreadsheet of all the services you know of that take donations? After that we can make it more interactive and get coding. :)

@mateoclarke is this information something the city website might be interested in hosting?

**Comment by vpruthi** [comment]
For sure! How/where/who should I share the google doc with?

Thanks!!

**Comment by werdnanoslen** [comment]
Could you please make it open so that anyone can edit, and then post the
link here?

**Comment by vpruthi** [comment]
Yes will do!

**Comment by werdnanoslen** [comment]
Hi again, will you be able to join our next meetup? Before that, you might drum up some support by posting on Slack, too :)

**Comment by anu007** [comment]
Hello, I am Anu and have a degree in UX Design.I wanted to join the team, Please let me know do you guys need UX Designer(Researcher)?I am open to work on other projects.Thanks.

**Comment by LillianChernin** [comment]
Hi @vpruthi!  My name is Lillian, I'm a dev (in training) and would love to work on this project.  Please let me know what I can do to help!

**Comment by vpruthi** [comment]
Wonderful let's do it! Sorry for the delay Anu and Lillian I recently had a
baby so just getting back to life :) Lillian and Anu do you want to send me
your e-mail addresses and we can find a time to chat? I am at
virgiliakpruthi@gmail.com

Or if there is another way you prefer let me know. We can also chat over
the phone 254 459 9599.
